src/applications/disability-benefits/all-claims/content/summaryOfEvidence.jsx

```diff
@@ -38,40 +60,48 @@ export const summaryOfEvidenceDescription = ({ formData }) => {
     );
   }
 
-  if (privateEvidence.length) {
-    const privateEvidenceList = privateEvidence.map(upload => (
+  if (
+    privateEvidenceUploads.length &&
+    privateEvidenceSelected &&
+    uploadPrivateEvidenceSelected
+  ) {
+    const privateEvidenceUploadsList = privateEvidenceUploads.map(upload => (
       <li key={upload.name}>{upload.name}</li>
     ));
     privateContent = (
       <div>
         <p>We’ll submit the below private medical records you uploaded:</p>
-        <ul>{privateEvidenceList}</ul>
+        <ul>{privateEvidenceUploadsList}</ul>
       </div>
     );
   }
 
-  if (layEvidence.length) {
-    const layEvidenceList = layEvidence.map(upload => (
-      <li key={upload.name}>{upload.name}</li>
+  if (
+    privateEvidence.length &&
+    privateEvidenceSelected &&
+    !uploadPrivateEvidenceSelected
+  ) {
+    const privateEvidenceList = privateEvidence.map(facility => (
+      <li key={facility.providerFacilityName}>
+        {facility.providerFacilityName}
+      </li>
     ));
-    layContent = (
+    privateEvidenceContent = (
       <div>
-        <p>We’ll submit the below supporting evidence you uploaded:</p>
-        <ul>{layEvidenceList}</ul>
+        <p>We'll get your private medical records from:</p>
```

Comment:
:robot: :rotating_light: `REGRESSION ALERT` :rotating_light: :robot: 

The straight apostrophe strikes again!

Actually, it looks like this is just the content that was already there, but re-ordered.

Either way, it should be a `’`. :man_shrugging:

Reply:
💯